From 5b991b814e9301360ed2f7338a466df46b422b43 Mon Sep 17 00:00:00 2001
From: "Patil, Rachna" <rachna@ti.com>
Date: Thu, 26 Apr 2012 16:23:54 +0530
Subject: [PATCH 219/609] ARM: OMAP: AM33XX: remove support for TSC alpha EVM

Original commit: 276c2ab072ca6beff237c6b4211b0c5ba2a7b796

  From git://arago-project.org/git/projects/linux-am33x.git
  And ti-sdk-am335x-evm-05.05.00.00-Linux-x86-Install image

Analog input connections to touchscreen were interchanged
on alpha evm's. Since we do not support this anymore,
this code is dropped.

Signed-off-by: Patil, Rachna <rachna@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index 8a86dca..3f73ff3 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -926,15 +926,7 @@ static void tsc_init(int evm_id, int profile)
 {
 	int err;
 
-	if (gp_evm_revision == GP_EVM_REV_IS_1_1A) {
-		am335x_touchscreen_data.analog_input = 1;
-		pr_info("TSC connected to beta GP EVM\n");
-	} else {
-		am335x_touchscreen_data.analog_input = 0;
-		pr_info("TSC connected to alpha GP EVM\n");
-	}
 	setup_pin_mux(tsc_pin_mux);
-
 	err = am33xx_register_tsc(&am335x_touchscreen_data);
 	if (err)
 		pr_err("failed to register touchscreen device\n");
-- 
1.7.5.4

